internal/server/jobs/jobs.go

```diff
@@ -0,0 +1,20 @@
+package jobs
+
+import (
+	"context"
+	"time"
+
+	"github.com/infrahq/infra/internal/server/data"
+)
+
+func RemoveOldDeviceFlowRequests(ctx context.Context, tx data.WriteTxn, lastRunAt, currentTime time.Time) error {
+	return data.DeleteExpiredDeviceFlowAuthRequests(tx)
+}
+
+func RemoveExpiredAccessKeys(ctx context.Context, tx data.WriteTxn, lastRunAt, currentTime time.Time) error {
```

Comment:
Seems like most of these arguments aren't used

Reply:
if you want more complex scheduling you may need them.

Reply:
Maybe, or maybe not, we don't know. If we need them we can add them. Maybe we'll need something totally different. It's a mistake to try and guess and what we need before we need it.